packages/management-api/src/index.ts

```diff
@@ -15,5 +15,6 @@ export async function isMgmtApiRegistered(): Promise<boolean> {
   return await registerManagementApi()
 }
 
-export { MgmtActions, log } from './globals'
 export { ManagedPod } from './managed-pod'
+export { ManagedProject } from './managed-project'
+export * from './globals'
```

Comment:
maybe we should not export everything from globals, as elements such as pluginName and log are not relevant in other contexts. Only export the required types.